x-pack/plugins/infra/public/alerting/metric_threshold/index.ts

```diff
@@ -30,5 +30,7 @@ Reason:
 `,
       }
     ),
+    isEditable: false,
```

Comment:
This metrics one can also be changed to `true` (thanks for changing the other two).